library/src/test/java/io/ona/kujaku/services/MapboxOfflineDownloaderServiceTest.java

```diff
@@ -419,6 +420,54 @@ public void getTaskStatusShouldUpdateCurrentDownloadMapNameWhenGivenValidDeleteQ
         assertEquals(expectedMapName, (String) getValueInPrivateField(mapboxOfflineDownloaderService, "currentMapDownloadName"));
     }
 
+    @Test
+    public synchronized void onStatusChangedShouldShowDownloadCompleteNotificationWhenGivenCompletedOfflineRegion() throws Throwable {
+        latch = new CountDownLatch(1);
+        OfflineRegionStatus completeOfflineRegionStatus = createOfflineRegion(OfflineRegion.STATE_ACTIVE, 300, 98923, 898, 230909, 300, true, true);
+
+        // Create dummy download task & insert it into the service
+        Intent sampleServiceIntent = createMapboxOfflineDownloaderServiceIntent();
+        sampleServiceIntent = createSampleDownloadIntent(sampleServiceIntent);
+
+        String mapName = sampleServiceIntent.getStringExtra(Constants.PARCELABLE_KEY_MAP_UNIQUE_NAME);
+
+        RealmDatabase realmDatabase = RealmDatabase.init(context);
+        insertValueInPrivateField(mapboxOfflineDownloaderService, "realmDatabase", realmDatabase);
+
+        mapboxOfflineDownloaderService.persistOfflineMapTask(sampleServiceIntent);
+
+
+        //MapBoxOfflineQueueTask mapBoxOfflineQueueTask = getTask(mapName);
+        //offlineQueueTasks.add(mapBoxOfflineQueueTask);
+
+        //insertValueInPrivateField(mapboxOfflineDownloaderService, "currentMapBoxTask", mapBoxOfflineQueueTask);
```

Comment:
Do we need to keep the commented-out code around?

Reply:
Removed